spec/models/partner_spec.rb

```diff
@@ -1,210 +0,0 @@
-# == Schema Information
```

Comment:
🤔 should we delete this whole file? Is it because you moved create and invite logic into those service objects?

Reply:
Oops this is an acident! Thanks for catching this